test/unit_tests/braket/circuits/test_qubit.py

```diff
@@ -28,7 +28,7 @@ def test_index_lt_zero():
 
 @pytest.mark.xfail(raises=TypeError)
 def test_index_non_int():
-    Qubit("not a number")
+    Qubit(0.5)
```

Comment:
```suggestion
    Qubit(qubit_arg)
```

Reply:
Sorry, wasn't sure how to do a multi-line suggestion lol

Reply:
You can drag down, just as you would for a comment,